ui/app/src/env/i18n-legacy.ts

```diff
@@ -1118,274 +1118,274 @@ export const controlsCommonMessages = defineMessages({
-  mg: { id: 'languages.mg', defaultMessage: 'Malagasy' },
-  mh: { id: 'languages.mh', defaultMessage: 'Marshallese' },
-  ms: { id: 'languages.ms', defaultMessage: 'Malay' },
-  na: { id: 'languages.na', defaultMessage: 'Nauru' },
-  nb: { id: 'languages.nb', defaultMessage: 'Bokmål, Norwegian; Norwegian Bokmål' },
-  nd: { id: 'languages.nd', defaultMessage: 'Ndebele, North; North Ndebele' },
-  ng: { id: 'languages.ng', defaultMessage: 'Ndonga' },
-  nl: { id: 'languages.nl', defaultMessage: 'Dutch; Flemish' },
-  nn: { id: 'languages.nn', defaultMessage: 'Norwegian Nynorsk; Nynorsk, Norwegian' },
-  no: { id: 'languages.no', defaultMessage: 'Norwegian' },
-  nr: { id: 'languages.nr', defaultMessage: 'Ndebele, South; South Ndebele' },
-  nv: { id: 'languages.nv', defaultMessage: 'Navajo; Navaho' },
-  ny: { id: 'languages.ny', defaultMessage: 'Chichewa; Chewa; Nyanja' },
-  oc: { id: 'languages.oc', defaultMessage: 'Occitan (post 1500)' },
-  oj: { id: 'languages.oj', defaultMessage: 'Ojibwa' },
-  om: { id: 'languages.om', defaultMessage: 'Oromo' },
-  os: { id: 'languages.os', defaultMessage: 'Ossetian; Ossetic' },
-  pi: { id: 'languages.pi', defaultMessage: 'Pali' },
-  ps: { id: 'languages.ps', defaultMessage: 'Pushto; Pashto' },
-  pt: { id: 'languages.pt', defaultMessage: 'Portuguese' },
-  qu: { id: 'languages.qu', defaultMessage: 'Quechua' },
-  rn: { id: 'languages.rn', defaultMessage: 'Rundi' },
-  rw: { id: 'languages.rw', defaultMessage: 'Kinyarwanda' },
-  sc: { id: 'languages.sc', defaultMessage: 'Sardinian' },
-  se: { id: 'languages.se', defaultMessage: 'Northern Sami' },
-  sg: { id: 'languages.sg', defaultMessage: 'Sango' },
-  sm: { id: 'languages.sm', defaultMessage: 'Samoan' },
-  sn: { id: 'languages.sn', defaultMessage: 'Shona' },
-  sr: { id: 'languages.sr', defaultMessage: 'Serbian' },
-  ss: { id: 'languages.ss', defaultMessage: 'Swati' },
-  st: { id: 'languages.st', defaultMessage: 'Sotho, Southern' },
-  su: { id: 'languages.su', defaultMessage: 'Sundanese' },
-  sv: { id: 'languages.sv', defaultMessage: 'Swedish' },
-  ti: { id: 'languages.ti', defaultMessage: 'Tigrinya' },
-  tl: { id: 'languages.tl', defaultMessage: 'Tagalog' },
-  to: { id: 'languages.to', defaultMessage: 'Tonga (Tonga Islands)' },
-  tw: { id: 'languages.tw', defaultMessage: 'Twi' },
-  ty: { id: 'languages.ty', defaultMessage: 'Tahitian' },
-  ug: { id: 'languages.ug', defaultMessage: 'Uighur; Uyghur' },
-  uz: { id: 'languages.uz', defaultMessage: 'Uzbek' },
-  ve: { id: 'languages.ve', defaultMessage: 'Venda' },
-  vo: { id: 'languages.vo', defaultMessage: 'Volapük' },
-  wa: { id: 'languages.wa', defaultMessage: 'Walloon' },
-  wo: { id: 'languages.wo', defaultMessage: 'Wolof' },
-  yo: { id: 'languages.yo', defaultMessage: 'Yoruba' },
-  za: { id: 'languages.za', defaultMessage: 'Zhuang; Chuang' },
-  zh: { id: 'languages.zh', defaultMessage: 'Chinese' }
-});
+export const languages = {
```

Comment:
Where is this used? Make a full inventory so we can decided where it needs to be moved to.

Reply:
languages are being used in 2 places right now, PathNavigator (it may end up removed) and locale-selector control. I was thinking on moving languages from i18n-legacy to utils/languages.ts, and `locale-selector` to use it as `craftercms.utils.languages`

Reply:
I don't want it bundled if it's just for FE1. Let's put it directly in the FE1 control/ds if it's just used in a single place.